heroku-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+if [ $APP_ENV == 'local' ] || [ $APP_ENV == 'staging'  ] ; then
+    composer install
+    php artisan migrate:fresh --seed
+else
+    php artisan migrate --force
+fi
```

Comment:
Was this added because of the node-notifier issue?

Reply:
Unrelated, this is needed to be able to seed databases